src/System.Runtime.InteropServices/src/ApiCompatBaseline.net463.txt

```diff
@@ -0,0 +1,4 @@
+MembersMustExist : Member 'System.Runtime.InteropServices.Marshal.PtrToStringUTF8(System.IntPtr)' does not exist in the implementation but it does exist in the contract.
```

Comment:
Make sure you have an issue tracking adding these to desktop and removing the baseline.